data/json/effects.json

```diff
@@ -4799,6 +4799,23 @@
       { "vitamin": "redcells", "rate": [ [ -250, -650 ] ], "tick": [ "1 s" ] }
     ]
   },
+  {
+    "id": "star_vampire_blood_drink",
```

Comment:
There is already an effect with same id, so this entry would overwrite it. This needs to have a different id or previously existing entry should be removed.